### PR TITLE
docs(action): deprecate the slot tooltip

### PR DESCRIPTION
**Pull request by josercarcamo**
**Related Issue:** #6371

## Summary
Deprecated slot tooltip

### CODE REVIEW

File: packages/calcite-components/src/components/action/action.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ import { CSS, SLOTS } from "./resources";
 
 /**
  * @slot - A slot for adding a `calcite-icon`.
- * @slot tooltip - A slot for adding a `calcite-tooltip`.
+ * @slot tooltip - [Deprecated] use `id` instead. A reference to a `calcite-tooltip`.
```
Comment (jcfranco) [inline packages/calcite-components/src/components/action/action.tsx:45]: The deprecation message should suggest using a tooltip instead. +@geospatialem @DitwanP for doc guidance.

**Review by jcfranco — CHANGES_REQUESTED** [review]
Since this is merely updating the doc, this should use the `docs` commit type. See https://github.com/ionic-team/stencil/blob/main/CONTRIBUTING.md#type for reference (referenced in our contributing doc).

File: packages/calcite-components/src/components/action/action.tsx (hunk 1 above)
Comment (geospatialem) [reply packages/calcite-components/src/components/action/action.tsx:45]: WDYT about the following for folks? cc: @DitwanP

 ```suggestion
 * @slot tooltip - [Deprecated] Use the `calcite-tooltip` component instead.
```

**Review by jcfranco — APPROVED** [review]
Doc changes look good! Once the PR title is updated to use the [`docs` type (vs `fix`)](https://github.com/Esri/calcite-design-system/pull/9270#pullrequestreview-2041861379), this should be good to merge!

**Pull request merged**